Scripts/Seleccionar.js

```diff
@@ -0,0 +1,12 @@
+/*import { redirigir, objeto } from './Modificar.js';*/
+
+function cargaIframe(Pagina) {
+    const Seleccion = document.querySelectorAll("#root");
+    let ventana = `<iframe src="${Pagina}.html" frameborder="0"></iframe>`;
+    document.getElementById("root").innerHTML = ventana;
+}
+var objeto1 = new Object('./Modificar.js');

```

Comment:
Codacy has a fix for the issue: [Replace `'./Modificar.js');␍` with `"./Modificar.js");`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456241)
```suggestion
var objeto1 = new Object("./Modificar.js");
```